src/components/title/title.jsx

```diff
@@ -0,0 +1,93 @@
+import { cn } from '../../utilities/functions';
+
+/**
+ * Title component.
+ */
+
+const Title = ( {
+	title = null,
+	description = null,
+	icon = null,
+	iconPosition = 'right', // left, right
+	tag = 'h2', // h1, h2, h3, h4, h5, h6
+	size = 'sm', // xs, sm, md, lg
+	className = '',
+} ) => {
+	// Base classes. - Mandatory classes.
+	const iconGap = {
+		xs: 'flex items-center gap-1',
```

Comment:
```suggestion
		xs: 'gap-1',
```
Since this part `flex items-center` is common for every icon size we can directly apply this in the classNames and keep the gap class name in the variable.

Reply:
Done